controllers/user.js

```diff
@@ -19,12 +19,13 @@ async function add(req, res) {
   const unverifiedSubscriber = await DB.addSubscriberUnverifiedEmailHash(email, fxNewsletter);
   const verifyUrl = EmailUtils.verifyUrl(unverifiedSubscriber);
   const unsubscribeUrl = EmailUtils.unsubscribeUrl(unverifiedSubscriber);
+  const serverUrl = req.app.locals.SERVER_URL;
 
   await EmailUtils.sendEmail(
     email,
     "Verify your subscription to Firefox Monitor.",
     "email_verify",
-    { email, verifyUrl, unsubscribeUrl}
+    { email, verifyUrl, unsubscribeUrl, serverUrl }
```

Comment:
Are you sick of this PR, or can we change this to `SERVER_URL` so we're consistent with the context variable name across email & website templates?

Reply:
the best thing about this PR is clicko expando so in homage to that, clicko disappeardo!